conf/modules.config

```
@@ -322,6 +322,49 @@ process {
         ]
     }
 
+    withName: SAMTOOLS_MERGE_LANES_BAMINPUT {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.prefix = { "${meta.sample_id}_${meta.library_id}_${meta.reference}" }
+        publishDir = [
+            enabled: false
+        ]
+        ext.args = { params.run_fastq_sharding ? "-c -p" : "" }
+    }
```

```suggestion
    withName: SAMTOOLS_MERGE_LANES_BAMINPUT {
        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
        ext.args = { params.run_fastq_sharding ? "-c -p" : "" }
        ext.prefix = { "${meta.sample_id}_${meta.library_id}_${meta.reference}" }
        publishDir = [
            enabled: false
        ]
    }
```
To keep order of code conventions